README.md

```diff
@@ -147,6 +147,8 @@ interface IBaseParseConfig {
   default?: string | number;
   // overrides FixedWidthParser.defaults.truncate
   truncate?: boolean;
+  // value to return when value parses as falsy
+  falsyFallback?: FalsyFallback
```

Comment:
We should make the documentation more explicit:

```suggestion
  // value to return if the parsed value is falsy
  // default: IParseOptions.falsyFallback value
  falsyFallback?: 'undefined' | 'null' | 'passthrough'
```

Reply:
We should also update the documentation up on L67 to explicitly document what the default value is for `IOptions.falsyFallback`. The default value is `'passthrough'`.